# message type that corresponds to inchworm.py
Header header
int8 id
int8[] ee1_pos
int8[] ee2_pos
int8 ee1_status
int8 ee2_status
int8 ee1_shingle_stat
int8 ee2_shingle_stat
int8 behavior
int8[] roof
int8 roof_width
int8[] shingle_depot_pos
int8[] ee1_valid_neighbors
int8[] ee2_valid_neighbors